### PR TITLE
Added new table-view style

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR adds a new "light" style for the datagrid table-view.

### CODE REVIEW

File: scss/modules/datagrid-table-view-light.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            tr.selected {
+                border-left: none;
+            }
+
+            .row.even, .row.odd {
+                background: transparent;
+            }
+        }
+
+        .task-state {
```
Comment (danrot) [inline scss/modules/datagrid-table-view-light.scss]: Should this table know about `tasks`?

Comment (wachterjohannes) [inline scss/modules/datagrid-table-view-light.scss]: i have overseen this. i will extract it to sulu

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
@wachterjohannes Please use your own fork the next time.

**Pull request merged**